chaosblade-exec-common/src/tests/java/com/alibaba/chaosblade/exec/common/util/StringUtilsTest.java

```diff
@@ -0,0 +1,226 @@
+package com.alibaba.chaosblade.exec.common.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StringUtilsTest {
+
+    @org.junit.Test
+    public void testHasLength() {
+        Assert.assertTrue(StringUtils.hasLength("foo"));
+
+        Assert.assertFalse(StringUtils.hasLength(""));
+    }
+
+    @org.junit.Test
+    public void testHasText() {
+        Assert.assertFalse(StringUtils.hasText(" "));
+        Assert.assertFalse(StringUtils.hasText(null));
+
+        Assert.assertTrue(StringUtils.hasText("foo"));
+    }
+
+    @org.junit.Test
+    public void testContainsWhitespace() {
+        Assert.assertFalse(StringUtils.containsWhitespace(null));
+        Assert.assertFalse(StringUtils.containsWhitespace("foo"));
+
+        Assert.assertTrue(StringUtils.containsWhitespace(" "));
+    }
+
+    @Test
+    public void testTrimWhitespace() {
+        Assert.assertNull(StringUtils.trimWhitespace(null));
+
+        Assert.assertEquals("foo", StringUtils.trimWhitespace(" foo "));
+    }
+
+    @Test
+    public void testTrimLeadingWhitespace() {
+        Assert.assertNull(StringUtils.trimLeadingWhitespace(null));
+
+        Assert.assertEquals("foo ", StringUtils.trimLeadingWhitespace(" foo "));
+    }
+
+    @Test
+    public void testTrimLeadingCharacter() {
+        Assert.assertNull(StringUtils.trimLeadingCharacter(null, 'a'));
+
+        Assert.assertEquals("oo",
+                StringUtils.trimLeadingCharacter("foo", 'f'));
+    }
+
+    @Test
+    public void testTrimTrailingWhitespace() {
+        Assert.assertNull(StringUtils.trimTrailingWhitespace(null));
+
+        Assert.assertEquals("foo",
+                StringUtils.trimTrailingWhitespace("foo "));
+    }
+
+    @Test
+    public void testTrimAllWhitespace() {
+        Assert.assertNull(StringUtils.trimAllWhitespace(null));
+
+        Assert.assertEquals("foo",
+                StringUtils.trimAllWhitespace(" f oo "));
+    }
+
+    @Test
+    public void testStartsWithIgnoreCase() {
+        Assert.assertFalse(StringUtils.startsWithIgnoreCase(null, null));
+        Assert.assertFalse(StringUtils.startsWithIgnoreCase("foo", "foobar"));
+        Assert.assertFalse(StringUtils.startsWithIgnoreCase("foo", "bar"));
+
+        Assert.assertTrue(StringUtils.startsWithIgnoreCase("foo", "f"));
+    }
+
+    @Test
+    public void testEndsWithIgnoreCase() {
+        Assert.assertFalse(StringUtils.endsWithIgnoreCase(null, null));
+        Assert.assertFalse(StringUtils.endsWithIgnoreCase("foo", "foobar"));
+        Assert.assertFalse(StringUtils.endsWithIgnoreCase("foo", "bar"));
+
+        Assert.assertTrue(StringUtils.endsWithIgnoreCase("foo", "o"));
+    }
+
+    @Test
+    public void testCountOccurrencesOf() {
+        Assert.assertEquals(0, StringUtils.countOccurrencesOf(null, null));
+        Assert.assertEquals(2, StringUtils.countOccurrencesOf("foo", "o"));
+    }
+
+    @Test
+    public void testReplace() {
```

Comment:
Is it possible to create a third test here for the third return in the method?

Reply:
Method called by a different method